### PR TITLE
Reformatted llogcolor with "black -S -l 79"

### PR DESCRIPTION
**Pull request by defaziogiancarlo**
This commit does a whole-file reformatting in a python3
style using the tool black, however the code is
semantically unchanged and is still python2.
This is a first step in porting to python3 and the
purpose is to separate most of the formatting changes
from the actual code changes.

### CODE REVIEW

File: scripts/llogcolor [hunk 1]
```diff
@@ -30,58 +30,72 @@ import subprocess
...
-
-            'on_black'      :    "\033[40m", 
-            'on_red'        :    "\033[41m",
-            'on_green'      :    "\033[42m",
-            'on_yellow'     :    "\033[43m",
-            'on_blue'       :    "\033[44m",
-            'on_magenta'    :    "\033[45m",
-            'on_cyan'       :    "\033[46m",
-            'on_white'      :    "\033[47m" }
+colors = {
```
Comment (ofaaland) [inline scripts/llogcolor:33]: In the earlier iteration of this patch, PR #23 , you mentioned that the python style guide isn't explicit about aligning (or not) the values when a series of KV pairs are given in the declaration of a dict.  Is there a good LC person or two, who've been involved in the effort to standardize our python scripts, to confirm they are using this unaligned convention as well?

**Comment by defaziogiancarlo** [comment]
To address some comments from the previous pull request I accidentally ended:

The code is being formatted by a tool that has different version of python available, but all are 3.X, so the code was formatted to some kind of python3 style. These changes aren't required to make the code work, but I'm considering this code to be kinda like new code, which should be in the python3 style per the LC python style guide.

I plan to use this code style consistently for all the python scripts that get ported. One of the places where the reformatting is most extreme is the parser code. Currently I'm planning to give all the scripts a parser using `argparse`. Right now it's a mix of `optparse`, `argparse`, and using `sys.argv` directly. This means that some of the scripts will get new parsers, and the ones that already use `argparse` will be get the same formatting as the scripts that get new code.

**Review by ofaaland — COMMENTED** [review]
Looks good!  Once we've got an answer as to whether to align values when declaring dicts, I'll approve.

File: scripts/llogcolor (hunk 1 above)
Comment (ofaaland) [reply scripts/llogcolor:33]: Several LC people responded in our python group chat and essentially said that so far no one in the group has objected to the formatting style of ```black```.  That's good enough for me.

**Pull request merged**